lib/rules/no-control-regex.js

```diff
@@ -51,29 +51,47 @@ module.exports = {
             return null;
         }
 
+
+        const controlChar = /[\x00-\x1f]/g; // eslint-disable-line no-control-regex
+        const multipleSlashes = /\\+/g;
+        const multipleSlashesAtEnd = /\\+$/g;
```

Comment:
Sorry, you're right - I wasn't being clear. https://github.com/eslint/eslint/pull/6839/files#r73994452 is where the check is done (in master already) - I'm just moving the regex higher. In this line, `\\+` matches one or more backslashes, so that the groups can later be checked.